gms/client/src/main/java/com/linkedin/chart/client/Charts.java

```diff
@@ -144,6 +153,34 @@ public BrowseResult browse(@Nonnull String path, @Nullable Map<String, String> r
         return _client.sendRequest(requestBuilder.build()).getResponse().getEntity();
     }
 
+    /**
+     * Update an existing Chart
+     */
+    public void update(@Nonnull final ChartUrn urn, @Nonnull final Chart chart) throws RemoteInvocationException {
+        Request request = CHARTS_ACTION_REQUEST_BUILDERS.createRequest(urn, toSnapshot(chart, urn));
+        _client.sendRequest(request).getResponse();
```

Comment:
We may want to log if there's an error here... Or somehow provide feedback.. Not sure if the other impls have this but we may want to

Reply:
We can address this wholistically in a followup. If there's any error, there should be a RemoteInvocationException thrown AFAIK

Reply:
Makes sense to me. Is the `sendRequest` method not already going to throw the exception though?